packages/trpc/server/routers/viewer/workflows/activateEventType.handler.ts

```diff
@@ -103,10 +110,12 @@ export const activateEventTypeHandler = async ({ ctx, input }: ActivateEventType
       const isSmsReminderNumberRequired = eventTypeWorkflow.steps.some((step) => {
         return step.action === WorkflowActions.SMS_ATTENDEE && step.numberRequired;
       });
-      await upsertSmsReminderFieldForBooking({
-        workflowId,
-        isSmsReminderNumberRequired,
-        eventTypeId,
+      [eventTypeId].concat(userEventType.children.map((ch) => ch.id)).map(async (evTyId) => {
+        await upsertSmsReminderFieldForBooking({
```

Comment:
@CarinaWolli does this look good?

Reply:
Yes, that looks good and is actually a good point! We also need to do that when deactivating a workflow, in line 91 `removeSmsReminderFieldForBooking`

And also when activating/deactivating the MET in the workflow itself